README.md

```diff
@@ -85,60 +85,44 @@ This function will create a new `RoleBinding` that depends on the `ServiceAccoun
 ## Build
 
 ```bash
-make
+$ make
 ```
 
 ## Installation
 
-Kogito Operator is not available in the OperatorHub [yet](https://issues.jboss.org/browse/KOGITO-67), hence have to be installed manually on [OpenShift 4.x](#deploy-to-openshift-4.x-using-olm) or [OpenShift 3.11](#deploy-to-openshift-311-manually).
+Kogito Operator is not available in the OperatorHub [yet](https://issues.jboss.org/browse/KOGITO-67), hence have to be installed manually on [OpenShift 4.x](#deploy-to-openshift-4.x) or [OpenShift 3.11](#deploy-to-openshift-311-manually).
 
 You can also [run the operator locally](#running-locally) if you have the [requirements](#requirements) configured in your local machine.
 
-### Upload to a container registry
+### Deploy to OpenShift 4.x
 
-e.g.
+First make sure that the Kogito image stream is created in the cluster:
 
 ```bash
-docker push quay.io/kiegroup/kogito-cloud-operator:<version>
+$ oc apply -f https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml -n openshift
 ```
 
-### Deploy to OpenShift 4.x using OLM
-
-To install this operator on OpenShift 4 for end-to-end testing, make sure you have access to a quay.io account to create an application repository. Follow the [authentication](https://github.com/operator-framework/operator-courier/#authentication) instructions for Operator Courier to obtain an account token. This token is in the form of "basic XXXXXXXXX" and both words are required for the command.
-
-Push the operator bundle to your quay application repository as follows:
+Then create an entry in the OperatorHub catalog with:
 
 ```bash
-operator-courier push deploy/catalog_resources/courier/0.1.0 kiegroup kogitocloud-operator 0.1.0 "basic XXXXXXXXX"
+$ oc create -f deploy/catalog_resources/courier/kogitocloud-operatorsource.yaml
 ```
 
-If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. Notice that the push command does not overwrite an existing repository, and the bundle needs to be deleted before a new version can be built and uploaded. Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) to load your operator bundle in OpenShift.
+It will take a few minutes for the operator to become visible under the _OperatorHub_ section of the OpenShift console _Catalog_. The Operator can be easily found by filtering by _Kogito_ name.
 
-Note that the OpenShift cluster needs access to the created application. Make sure that the application is **public** or you have configured the private repository credentials in the cluster. To make the application public, go to your quay.io account, and in the _Applications_ tab look for the `kogitocloud-operator` application. Under the settings section click on _make public_ button.
+Verify if the Operator status by running:
```

Comment:
"Verity if the operator status" does not make sense, please work out the actual requirement. Verify operator status? Verify availability of the operator>

Reply:
I guess that "Verify operator status" sounds better, plus makes more sense for this context.